.semaphore/backend-deploy.yml

```diff
@@ -0,0 +1,52 @@
+version: v1.0
+name: JSgram CI
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+execution_time_limit:
+  minutes: 2
+
+
+blocks:
+  - name: BE Deploy
+    task:
+      prologue:
+        commands:
+          - checkout
+          - node --version
+          - npm --version
+          - git --version
+          - echo "machine git.heroku.com
+                    login ${HEROKU_USER}
+                    password ${HEROKU_PASSWORD}" >> $HOME/.netrc
+          - git config --global user.email "${GIT_EMAIL}"
+          - git config --global user.name "${GIT_USER}"
+      env_vars:
+        - name: GIT_EMAIL
+          value: vf.velt@gmail.com
+        - name: GIT_USER
+          value: vglfr
```

Comment:
is it correct to hard code your own email?

Reply:
One could find it in `git log` of the very commit. The principle you're referring to seem sound to me though; hence changes.